package.json

```diff
@@ -27,8 +27,10 @@
     "commitmsg": "validate-commit-msg",
     "build:spec:browser": "echo \"Browser test is not working currently\" && exit -1 && webpack --config spec/support/webpack.mocha.config.js",
     "decision_tree_widget": "cd doc/decision-tree-widget && npm run build && cd ../..",
-    "lint_spec": "tslint -c tslint.json \"spec/**/*.ts\"",
-    "lint_src": "tslint -c tslint.json \"src/**/*.ts\"",
+    "lint-spec": "tslint -c tslint.json \"spec/**/*.ts\"",
+    "lint_eslint_spec": "eslint --ext=ts spec/**/*  -c .eslintrc",
```

Comment:
I'd like to suggest to follow `:` for script convention (using `-` was old way we did), like `lint:eslint:spec`.

btw, maybe with eslint we can just make single lit to run all spec / src at once? I think it was splitted as tslint glob behaves weird on windows.

Reply:
Actually the tslint run scripts will be removed so I will do this in my next commit to give a good overview on what the change is.

Reply:
Comment was for new script `eslint`.